### ISSUE TITLE
Maven build fails when changing version from 1.7.9 to 1.8.0

### ISSUE THREAD

**Issue by jeffkreska** [wro4j/wro4j#1029]
It appears that the contextFolder option is not being honored anymore.

Here is the configuration section from my maven plugin.

```
<configuration>
    <jsDestinationFolder>${js.releaseDir}/${js.releaseName}/</jsDestinationFolder>
    <cssDestinationFolder>target/${project.artifactId}-${project.version}/${resource.path}/css/</cssDestinationFolder>
    <minimize>${minimize.resources}</minimize>
    <contextFolder>${basedir}/src/main/webapp/,${basedir}/../../,${basedir},${basedir}/src/main/sass/</contextFolder>
    <wroManagerFactory>ro.isdc.wro.maven.plugin.manager.factory.ConfigurableWroManagerFactory</wroManagerFactory>
    <wroFile>${basedir}/src/main/webapp/WEB-INF/wro.xml</wroFile>
    <extraConfigFile>${basedir}/src/main/webapp/WEB-INF/wro.properties</extraConfigFile>
</configuration>
```

**Comment by alexo** [comment]
Thanks for reporting. Will investigate it...

**Comment by sebint** [comment]
This issue seems like raised again in version 2.0.0.
Maven build work successfully with 1.10.1 but fails when wro4j-maven-plugin version changed to version 2.0.0.

`"contextFolder was not set!"`

Thats the error I get.
My Configuration is below;

```
        <configuration>
          <ignoreMissingResources>false</ignoreMissingResources>
          <contextFolder>${project.basedir}/src/main/resources/static</contextFolder>
          <cssDestinationFolder>${project.build.directory}/classes/static/assets/bundles</cssDestinationFolder>
          <jsDestinationFolder>${project.build.directory}/classes/static/assets/bundles</jsDestinationFolder>
          <wroFile>${project.basedir}/src/main/resources/wro/wro.xml</wroFile>
          <extraConfigFile>${project.basedir}/src/main/resources/wro/wro.properties</extraConfigFile>
          <wroManagerFactory>ro.isdc.wro.maven.plugin.manager.factory.ConfigurableWroManagerFactory</wroManagerFactory>
        </configuration>
```

**Comment by ppodgorsek** [comment]
This issue has been solved in a duplicate ticket (#1155).
